application/dash_application/known_pages_router.py

```diff
@@ -31,6 +31,8 @@ def menu_item(label, link):
     return dbc.NavbarSimple(
         id="navbar",
         children=[
+            # Simple Navbar only lets you add content to the right side. This CSS forces the button out of the div 
+            html.A(html.Button("Home", style={"position": "fixed", "left": "0", "margin-left": "15px", "margin-top": "4px"}), href="/.."),
```

Comment:
I think either `"/"` or `".."`, but both is not doing anything.

Reply:
Well that's weird, because it works for me. What's your environment like? I'm on firefox 76

Reply:
Oh, it works, it's just _go to the parent of the root directory_ doesn't mean anything beyond _go to the root directory_.

Reply:
Oh, didn't know a link to / would resolve, figured the link would be constructed like 'localhost:5000/single-lane-ts?run=my_cool_run/..'